frame/vault/src/models.rs

```diff
@@ -14,7 +14,12 @@ pub struct VaultInfo<AccountId, Balance, CurrencyId, BlockNumber> {
 
 #[derive(Copy, Clone, Encode, Decode, Default, Debug, PartialEq, TypeInfo)]
 pub struct StrategyOverview<Balance> {
-	/// The reported balance of the strategy
+	/// The reported balance of the strategy.
+	///
+	/// Added when withdrawn and subtracted when deposited more or less like a "borrowed" balance.
```

Comment:
added by via withdrawals from vault and substrate by depositign back to vault

Reply:
Changed